content/en/docs/reference/config/analysis/ist0108/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: UnknownAnnotation
+layout: analysis-message
+---
+
+This message occurs when you attach an annotation to a namespace, but Istio does
+not recognize the annotation.
+
+Istio only recognizes certain [annotation names](/docs/reference/config/annotations/).
+
+To resolve this problem, check the name of your annotation and try again.
```

Comment:
I believe the behavior here is to only check annotations that are named such that they appear to be *istio* annotations. @esnible, does that sound correct?

Reply:
Correct.  To trigger UnknownAnnotation, use an annotation matching `*istio.io[/*]` that we don't recognize.  (We have a hand-crafted copy of the annotations in istio.io/api/annotations), the same ones that appear at https://istio.io/docs/reference/config/annotations/

We don't complain if the user starts using unknown non-Istio annotation `halloween.io/pumpkin`.   Just `istio.io` annotations.

Reply:
fixed